hack/.golint_failures

```diff
@@ -558,6 +558,7 @@ staging/src/k8s.io/sample-apiserver/pkg/apis/wardle/v1alpha1
 staging/src/k8s.io/sample-apiserver/pkg/registry/wardle/fischer
 staging/src/k8s.io/sample-apiserver/pkg/registry/wardle/flunder
 test/e2e/common
+test/e2e/framework/metrics
```

Comment:
Please fix golint failures of the file instead of adding the exception.
We've taken much time for fixing them under `test/e2e/framework` already.

Reply:
As i mentioned in a previous comment, the golint failures we are ignoring here are from `MetricsForE2E`. We are open to any suggestion you may have to rename that struct but otherwise the plan is to ignore them for this package.